diff --git a/website/manage.py b/website/manage.py
index cba16e39..ce824f8d 100755
--- a/website/manage.py
+++ b/website/manage.py
@@ -8,6 +8,7 @@ import sys
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "thaliawebsite.settings")
+    os.environ.setdefault("ENABLE_LOGFILE", "0")
     try:
         from django.core.management import execute_from_command_line
     except ImportError as exc:
diff --git a/website/thaliawebsite/settings/production.py b/website/thaliawebsite/settings/production.py
index ff7ffabc..95e0f7b6 100644
--- a/website/thaliawebsite/settings/production.py
+++ b/website/thaliawebsite/settings/production.py
@@ -58,7 +58,7 @@ SENDFILE_URL = "/media/sendfile/"
 SENDFILE_ROOT = "/concrexit/media/"
 
 STATIC_URL = "/static/"
-STATIC_ROOT = "/concrexit/static"
+STATIC_ROOT = os.environ.get("STATIC_ROOT", "/concrexit/static")
 
 if not DEBUG:
     COMPRESS_OFFLINE = True
@@ -121,35 +121,36 @@ if os.environ.get("DJANGO_EMAIL_HOST"):
     EMAIL_USE_SSL = os.environ.get("DJANGO_EMAIL_USE_SSL", False) == "True"
     EMAIL_TIMEOUT = 10
 
+# Default logging: https://github.com/django/django/blob/master/django/utils/log.py
+# We disable mailing the admin.
+# Server errors will be sent to Sentry via the config below this.
 LOGGING = {
     "version": 1,
-    "disable_existing_loggers": True,
     "formatters": {
         "verbose": {"format": "%(asctime)s %(name)s %(levelname)s %(message)s"},
     },
     "handlers": {
         "console": {
-            "level": "WARNING",
-            "class": "logging.StreamHandler",
-            "formatter": "verbose",
-        },
-        "logfile": {
             "level": "INFO",
-            "class": "logging.FileHandler",
+            "class": "logging.StreamHandler",
             "formatter": "verbose",
-            "filename": "/concrexit/log/django.log",
-        },
+        }
     },
     "loggers": {
-        "django": {
-            "handlers": ["console", "logfile"],
-            "level": "INFO",
-            "propagate": False,
-        },
-        "": {"handlers": ["logfile"], "level": "INFO",},
+        "django": {"handlers": [], "level": "INFO"},
+        "root": {"handlers": ["console"], "level": "INFO"},
     },
 }
 
+if os.environ.get("ENABLE_LOGFILE", "1") == "1":
+    LOGGING["handlers"]["logfile"] = {
+        "level": "INFO",
+        "class": "logging.FileHandler",
+        "formatter": "verbose",
+        "filename": os.environ.get("LOGFILE", "/concrexit/log/django.log"),
+    }
+    LOGGING["loggers"]["root"]["handlers"] += ["logfile"]
+
 sentry_sdk.init(
     dsn=os.environ.get("SENTRY_DSN"),
     integrations=[DjangoIntegration()],
